docs/getting-started/package-infrastructure.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Package Infrastructure
 toc: true
-weight: 6
+weight: 3
 indent: true
 ---
 
 # Package Infrastructure
```

Comment:
I feel like what we package is not infrastructure, but templates for the infrastructure. In other words, `packaging infrastructure` would mean literally zipping the managed resource YAMLs for me whereas here what we package is `Composition`s and XRDs

Reply:
Hmm I see what you mean.. I am not sure what a good alternative is here, maybe something like `Package Your Platform`? I think the motivation for `Package Infrastructure` is packaging your infrastructure *APIs*. Maybe we could literally say `Package Infrastructure APIs`

Reply:
I think I named this to stick with the `<Verb> Infrastructure` theme. If we want to break that theme something like "Create a Configuration" or "Create a Composition" could make sense.